doc/source/datainputs.rst

```diff
@@ -46,28 +146,27 @@ Assuming knowledge of the box used in the simulation, a LAMMPS XYZ file could be
     traj = np.genfromtxt(
         'trajectory.xyz', skip_header=2,
         invalid_raise=False)[:, 1:4].reshape(-1, N, 3)
+    box = freud.box.Box.cube(L=20)
 
-    for frame in traj[frame_start:]:
-        rdf.compute((frame.configuration.box, frame.particles.position), reset=False)
+    for frame_positions in traj:
+        rdf.compute(system=(box, frame_positions), reset=False)
 
-Note that various readers do exist for XYZ files, but due to their simplicity we choose to read them in manually in this example.
 The first line is the number of particles, so we read this line and use it to determine how to reshape the contents of the rest of the file into a NumPy array.
 
-Using External Readers (MDAnalysis for DCD files)
-=================================================
+Other External Readers
+======================
 
 For many trajectory formats, high-quality readers already exist.
-Packages such as `MDAnalysis <https://www.mdanalysis.org/>`_, `MDTraj <http://mdtraj.org/>`_, `pytraj <https://amber-md.github.io/pytraj/latest/read_and_write.html>`_, or `garnett <https://garnett.readthedocs.io/>`_ can read virtually all common trajectory formats.
-
-DCD files are among the most familiar simulation outputs due to their longevity.
-Here, we provide an example using `MDAnalysis <https://www.mdanalysis.org/>`_ to read data from a DCD file.
+However sometimes these readers' data structures must be converted into a format understood by **freud**.
+Below, we show an example that converts the MDAnalysis box dimensions from a matrix into a :class:`freud.box.Box`.
+(Note that MDAnalysis inputs are natively supported by **freud** without this extra step.)
```

Comment:
Remove parenthesis, and add a quick ref to the section above where using MDAnalysis is demonstrated.